provider/terraform.rb

```diff
@@ -161,5 +161,25 @@ def generate_resource_tests(data)
         out_file: filepath
       )
     end
+
+    # Only going to be run once.
+    def generate_async(output_folder, _types, version_name)
```

Comment:
nit: `generate_operation` or `generate_operation_waiter` probably better indicates what we're doing here.